src/scylla/db.clj

```diff
@@ -439,6 +445,18 @@
     (when (cu/exists? disabled)
       (c/exec :mv disabled scylla-bin))))
 
+(defn wait-for-previous [test node]
+  (let [nodes (:nodes test)
+        fst (first nodes)]
+    (when (not= node fst)
+        (loop [[previous & tail] nodes]
+            (if (= node (first tail))
+                (do
+                  (info "Waiting for" previous)
+                  (sc/close! (sc/await-open test previous))
+                  (info previous "is available"))
+                (recur tail))))))
```

Comment:
It looks like you reimplemented `map` here.
Although for this use case, `run!` seems like a better candidate:
https://clojuredocs.org/clojure.core/run!

Reply:
Sorry, you actually didn't implement `map`.
You want to process an element IF the subsequent element satisfies a predicate. (if it's `node`)
So nevermind.